prompts.js

```diff
@@ -0,0 +1,25 @@
+console.log("In order to setup section you need to get your projectId and projectUrl by registering an account on https://sections.geeks.solutions")
+
+module.exports = [
+    {
+        type: 'input',
+        name: 'projectUrl',
+        message: 'What is your project url:',
+        validate: input => !!input,
+        default: ''
+    },
+    {
+        type: 'input',
+        name: 'projectId',
+        message: 'What is your project ID:',
+        validate: input => !!input,
+        default: ''
+    },
+    {
+        type: 'confirm',
+        name: 'addSectionsFiles',
+        message: 'Add vue-sections configuration with first sections page',
+        description: 'This will generate a ready to use sections page, vue-sections configuration files and add the required dependencies to your package.json',
```

Comment:
wait what ? this isn't what we agreed to...

The confirmation should be just about adding the page.

All the rest of the setup should happen!

Reply:
I forgot only to change the description, but the behavior is like described now.
 > The confirmation should be just about adding the page.